lib/ProductOpener/Tags.pm

```diff
@@ -311,6 +314,23 @@ sub get_property ($tagtype, $canon_tagid, $property) {
 	}
 }
 
+sub get_property_with_fallbacks ($tagtype, $tagid, $property, $fallback_lcs = ["xx", "en"]) {
+
+	my $property_value = get_property($tagtype, $tagid, $property);
+	if (!defined $property_value) {
+		# is it language dependent ?
+		if ($property =~ /:..$/) {
+			my $bare_name = $`;
+			# try fallbacks
+			foreach my $lang (@$fallback_lcs) {
```

Comment:
There's a $lang global in Lang.pm, it's best to use another name (otherwise there will be warnings)

Reply:
ok, changed for $lc.